lib/si-jwt-public-key/src/lib.rs

```diff
@@ -124,6 +126,13 @@ pub struct SiJwtClaimsV1 {
 }
 
 impl SiJwtClaims {
+    pub fn token_id(token: &SiJwt) -> JwtKeyResult<Option<AuthTokenId>> {
+        match token.jwt_id {
```

Comment:
you can wrap the whole match in Ok() and then you don't need an Ok in each match arm

Reply:
I go back and forth over which of these looks better! I can never decide :(